Form/Recaptcha3Type.php

```diff
@@ -8,7 +8,7 @@
 use Symfony\Component\Form\FormView;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class Recaptcha3Type extends AbstractType
+final class Recaptcha3Type extends AbstractType
```

Comment:
@karser  This is a serious BC break while you released a minor version. Be careful if a lot of people use your package it can break their application.

Reply:
@Nek- Thanks for noticing!